src/SystemCommands-MessageCommands/SycFindAndReplaceMethodCommand.class.st

```diff
@@ -0,0 +1,59 @@
+"
+I am a command to find and replace selector in a range of methods, with a call to itself
+"
+Class {
+	#name : #SycFindAndReplaceMethodCommand,
+	#superclass : #SycChangeMessageSignatureCommand,
```

Comment:
It does not look like a message related command because it is based on ast of the method. I think it should be a kind of SycMethodCommand (it's based on multiple methods but you can require single method in your class)